src/goto-symex/symex_complexity.cpp

```diff
@@ -0,0 +1,195 @@
+/*******************************************************************\
+
+Module: Symbolic Execution
+
+Author: John Dumbell
+
+\*******************************************************************/
+
+#include "symex_complexity.h"
+#include "goto_symex_state.h"
+#include <cmath>
+
+symex_complexityt::symex_complexityt(
+  message_handlert &logger,
+  const optionst &options)
+  : log(logger)
+{
+  std::size_t limit = options.get_signed_int_option("symex-complexity-limit");
+  if((complexity_active = limit > 0))
```

Comment:
weird parentheses

Reply:
Needed for the assignment, doesn't like it otherwise.